src/pages/nft/claim/index.vue

```diff
@@ -171,6 +169,15 @@ export default {
       );
     },
   },
+  watch: {
+    getAddress() {
+      this.claimingAddressInput = this.getAddress;
+    },
+    loginAddress() {
+      this.claimingAddressInput = this.loginAddress;
+      this.onEnterClaimingAddress();
```

Comment:
Should separate the logTrackEvent from this function?

Reply:
Updated to directly assign `claimingAddress`